fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergMetadata.java

```diff
@@ -904,7 +904,11 @@ public static PartitionData partitionDataFromPath(String relativePartitionPath,
                     field.name().equals(parts[0]), "Invalid partition: %s", partitions[i]);
 
             org.apache.iceberg.types.Type sourceType = spec.partitionType().fields().get(i).type();
-            data.set(i, Conversions.fromPartitionString(sourceType, parts[1]));
+            if (parts[1].equals("null")) {
```

Comment:
why we can insert null partition?  There are many points that need to be adapted if we support it.

Reply:
it is tested